guest-os-info.md

```diff
@@ -0,0 +1,35 @@
+## Guest Operating System Information
+
+Guest operating system identity for the Virtual Machine will be provided by the label ``os-short-id`` :
+
+```
+metadata:
+  name: myvm
+  labels:
+    os-short-id:
```

Comment:
I'd add an example value here - i.e. `win2k12r2` - and then use the example value in the `osinfo query` line below (to make the connection to the user).